### ISSUE TITLE
Warning thrown after upgrading to 6.7.0

### ISSUE THREAD

**Issue by thedewpoint** [wix/react-native-ui-lib#1766]
## Description

I upgraded to 6.7.0 from 6.1.2 in order to leverage the Toast component. My app works fine, but I'm getting a warning every time I start it (Android emulator) I'm using Expo just for additional context
```
Warning: forwardRef render functions do not support propTypes or defaultProps. Did you accidentally pass a React component?
```

### Related to

- [x ] Components

- React Native:
- React Native UI Lib:
- Expo

### Affected platforms

- [ x] Android

**Comment by thedewpoint** [comment]
I was able to resolve this by using package imports instead of just globally importing react-native-ui-lib 
i.e
```
import Toast from 'react-native-ui-lib/toast';
```
Not sure what the problem is but seems to be good now.

**Comment by yinshurman** [comment]
same issues with 6.7.1

**Comment by ethanshar** [comment]
This should have been fixed in 6.8.1
